### PR TITLE
Update NotGovUK to v0.3.2

### PR DESCRIPTION
**Pull request by daniel-ac-martin**
This updates the underlying framework and gives us some nice links to previews in PRs.

### CODE REVIEW

File: apps/docs/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "react-helmet-async": "^1.0.7",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "serverless": "^1.75.1",
+    "serverless": "^2.62.0",
```
Comment (RobinKnipe) [inline apps/docs/package.json:55]: Any chance of testing a ~possible~ _probable_ breaking change? Or can this only be done by Cypress tests running against the PR prototype URL?.. which, is that even setup yet!?

File: apps/docs/package.json [hunk 2]
```diff
@@ -9,15 +9,27 @@
...
     "build": "webpack",
     "package:serverless": "sls package",
+    "heroku-postbuild": "npm run build",
     "clean": "rm -rf .serverless dist",
     "dev": "NODE_ENV=development node-hot --fork | bunyan",
-    "dev:ssr": "NODE_ENV=development SSR_ONLY=true node-hot --fork | bunyan"
+    "dev:ssr": "NODE_ENV=development SSR_ONLY=true node-hot --fork | bunyan",
+    "create": "plop",
+    "create:deployment": "plop deployment",
+    "create:page": "plop page"
```
Comment (RobinKnipe) [inline apps/docs/package.json:18]: Should the plops be included here or can they just run from the upstream project via `npx`?
Should they only be included here if the upstream defaults have been changed to do something project specific?

File: apps/docs/plopfile.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = plop => {
+  plop.load('@not-govuk/app-plop-pack');
```
Comment (RobinKnipe) [inline apps/docs/plopfile.js:4]: With reference to my earlier comment: couldn't this file be replaced by a simple script in `package.json` like `"plop": "npx @not-govuk/app-plop-pack ..."` ?

File: .storybook/decorators.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { BrowserRouter as Router } from 'react-router-dom';
 const Root: FC<any> = props => (
   <div id="story-root" {...props} style={{
     backgroundColor: 'white',
-    fontFamily: [ 'GDS Transport', 'Roboto', 'Arial', 'sans-serif' ],
+    fontFamily: [ 'Roboto', 'GDS Transport', 'Arial', 'sans-serif' ],
```
Comment (RobinKnipe) [inline .storybook/decorators.tsx:8]: :+1:

File: apps/ui/src/server/index.ts [hunk 5]
```diff
@@ -42,12 +42,14 @@ const startApp = () => stage1.then(
       ( config.auth.method === AuthMethod.None && { method: AuthMethod.None } )
         || ( config.auth.method === AuthMethod.Dummy && { method: AuthMethod.Dummy, ...config.auth.dummy } )
```
[the remark this answers is not in the document]
Comment (daniel-ac-martin) [reply apps/ui/src/server/index.ts:43]: Hmm, not sure.

It's boilerplate as far as LEV is concerned though. The source of this is in NGU.

File: apps/docs/package.json (hunk 1 above)
Comment (daniel-ac-martin) [reply apps/docs/package.json:55]: It deploys to Netlify so it seems to be working. ;-)

File: apps/ui/src/server/config.ts [hunk 6]
```diff
@@ -48,6 +54,7 @@ const serverConfig = {
     port: Number(process.env.LISTEN_PORT) || 8080
   },
   mode: (process.env.MODE || 'server') as Mode,
+  privacy: !!(process.env.PRIVACY && process.env.PRIVACY.match(/(yes|true)/i)),
```
[the remark this answers is not in the document]
Comment (RobinKnipe) [reply apps/ui/src/server/config.ts:57]: :100:

**Review by RobinKnipe — APPROVED** [review]
Nice!

**Pull request merged**